CHANGELOG.md

```diff
@@ -0,0 +1,990 @@
+# mdn-bcd-collector Changelog
+
+## v3.3.1
+
+- (Automatically tag new releases (#1484), 2021-09-15)
+- (Add RTCSessionDescription test; fix RTCDTMFToneChangeEvent (#1488), 2021-09-15)
+- (Bump @google-cloud/storage from 5.14.1 to 5.14.2, 2021-09-14)
```

Comment:
Can you filter out all of the dependency upgrades, or perhaps summarize them as a single "Upgraded dependencies" item at the end of each release? (That would avoid the problem of an empty changelog if only deps changes.)

Reply:
I think it makes sense to state updates to the webref and BCD packages, but the rest is not useful to collector consumers.  I'll strip out all non-webref and non-BCD updates!